Dockerfile

```diff
@@ -39,4 +39,4 @@ WORKDIR /app
 COPY --from=builder /usr/local/bundle/ /usr/local/bundle/
 COPY --from=builder /app ./
 
-CMD bundle exec puma
+CMD GOVUK_PROMETHEUS_EXPORTER=true bundle exec puma
```

Comment:
I think this should work cos I believe it'll go through `sh` when it's in this form (as opposed to the list form) but definitely check that it works IRL if you haven't already!

Reply:
yeah, it does work... I checked yesterday that the metrics port was open on normal docker run and that you can run rake task without another prometheus exporter being set. I will do that in other places.